### ISSUE TITLE
Keyboard shortcut

### ISSUE THREAD

**Issue by huyz** [herdsothom/joplin-insert-date#2]
Right now, there's a keyboard shortcut to "Insert Date Time". Can you add one for "Insert Date"?

**Comment by petzi53** [comment]
I agree. A keyboard shortcut is essential. Without a keyboard shortcut, one has to interrupt typing to grab the mouse. Using the keyboard shortcut for "Insert Date Time" and deleting the time with several backspaces is more comfortable and faster for me.

**Comment by cwross** [comment]
I, too, would be grateful for a keyboard short option!

**Comment by knas4ever** [comment]
yes a option for setting a keyboard shortcut would be awesome. please, please, pretty please 😃
